src/assets/templates/widgets/activity.html

```diff
@@ -1,17 +1,33 @@
-<div class="cc-widget cc-widget-activities" data-ng-controller="ActivityController">
+<div
```

Comment:
@developish - It appears that we're no longer using `widgets/activity.html`, should we remove this template?

Reply:
Yes